.github/workflows/dashboards-ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Dashboards CI
```

Comment:
same here, highlevel rationale is alwasy appreciate as comment

check json on pr, build only on master